server_extensions_api/src/main/java/com/generalbytes/batm/server/extensions/IExtensionContext.java

```diff
@@ -264,6 +278,26 @@ IIdentity updateIdentity(String identityId, String externalId, int state, int ty
                              List<ILimit> limitCashPerMonth, List<ILimit> limitCashPer3Months, List<ILimit> limitCashPer12Months, List<ILimit> limitCashPerCalendarQuarter,
                              List<ILimit> limitCashPerCalendarYear, List<ILimit> limitCashTotalIdentity, String configurationCashCurrency);
 
+    /**
+     * @param customFieldDefinitionId use {@link CustomFieldDefinition#getId()} of a custom field to set
+     */
+    void setIdentityCustomField(String identityPublicId,
+                                long customFieldDefinitionId,
+                                CustomFieldValue customFieldValue) throws RuntimeException;
```

Comment:
TODO check types, try to avoid setting e.g. a string value to a checkbox custom field

Reply:
added subtypes for CustomFieldValue, 
but if it matches the customFieldDefinition type will be still checked at runtime